src/main/java/com/work/management/entity/Employee.java

```diff
@@ -0,0 +1,50 @@
+package com.work.management.entity;
+
+import lombok.*;
+import javax.persistence.*;
+import java.util.Date;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "EMPLOYEES")
+public final class Employee {
+
+    @Column(name = "firstName")
+    private String firstName;
+
+    @Column(name = "lastName")
+    private String lastName;
+
+    @Column(name = "userName")
+    private String userName;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id")
+    private Integer id;
+
+    @Column(name = "manager")
+    private Integer manager;
+
+    @Column(name = "teamId")
+    private String teamId;
+
+    @Column(name = "phoneNumber")
+    private String phoneNumber;
+
+    @Column(name = "createdAtTimeStamp")
+    private Date createdAtTimeStamp;
+
+    @Column(name = "lastUpdatedTimeStamp")
+    private Date lastUpdatedTimeStamp;
+
```

Comment:
Use @CreatedDate and @LastModifiedDate on createdAtTimeStamp and lastUpdatedTimeStamp respectively.

https://stackoverflow.com/questions/49954812/how-can-you-make-a-created-at-column-generate-the-creation-date-time-automatical